Tests/Find-ImageName.Tests.ps1

```diff
@@ -1,13 +1,101 @@
 Import-Module -Force $PSScriptRoot/../Docker.Build.psm1
 
+function Get-TempPath {
```

Comment:
I suggest moving the functions here to a file of their own. They do not belong next to the actual tests in Describe, Context and It blocks.

Reply:
Moved to its own file, `Utilities.ps1` + its tests.